apps/spruce/src/gql/client/link/logGQLErrorsLink.ts

```diff
@@ -11,7 +12,20 @@ export const reportingFn =
     if (path) {
       fingerprint.push(...path);
     }
-    reportError(new Error(gqlErr.message), {
+
+    const isValidationError =
+      gqlErr.extensions?.code ===
+      ApolloServerErrorCode.GRAPHQL_VALIDATION_FAILED;
+
+    const err = isValidationError
+      ? new GraphQLError(
+          `GraphQL validation error in '${operation.operationName}': ${gqlErr.message}`,
+        )
+      : new Error(
```

Comment:
Why not utilize `GraphQLError` instead of `Error`?

Reply:
I was thinking `GraphQLError` would be like a legitimate error in GraphQL, whereas `Error` might be an error in our actual application! So we might get an error because of wrong implementation in Evergreen, but it doesn't mean that something is wrong in GraphQL itself